### PR TITLE
Frontend lint

### PR DESCRIPTION
**Pull request by La0**
Needs #143 to be merged first

### CODE REVIEW

File: frontend/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+env:
+  browser: true
+  es6: true
+extends:
+  - google
```
Comment (marco-c) [inline frontend/.eslintrc.yml]: Can we use the mozilla rules?

File: frontend/.eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+globals:
+  Atomics: readonly
+  SharedArrayBuffer: readonly
+parserOptions:
+  ecmaVersion: 2018
+  sourceType: module
+rules:
+  require-jsdoc: off
+  max-len: [warn, {code: 120}]
```
Comment (marco-c) [inline frontend/.eslintrc.yml]: Nit: I'd disable this altogether

File: frontend/.eslintrc.yml (hunk 1 above)
Comment (La0) [reply frontend/.eslintrc.yml]: Yeap :+1:

File: frontend/.eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  browser: true
+  es6: true
+plugins:
+  - prettier
+  - mozilla
+extends:
+  - standard
+  - prettier
+  - plugin:mozilla/recommended
+globals:
```
Comment (marco-c) [inline frontend/.eslintrc.yml]: Are these necessary?

File: frontend/src/common.js [hunk 4]
```diff
@@ -209,7 +210,7 @@ export async function getSource(file) {
   const response = await fetch(
     `https://hg.mozilla.org/mozilla-central/raw-file/tip/${file}`
   );
-  return await response.text();
+  return response.text();
```
Comment (marco-c) [inline frontend/src/common.js]: Don't we need to await here?

File: frontend/src/zero_coverage_report.js [hunk 5]
```diff
@@ -125,15 +125,6 @@ function cumStats(prevStats, newStats) {
   );
 }
 
-function getFileSize(size) {
```
Comment (marco-c) [inline frontend/src/zero_coverage_report.js]: It was unused?

File: frontend/src/common.js (hunk 4 above)
Comment (La0) [reply frontend/src/common.js]: Come on Marco, https://eslint.org/docs/rules/no-return-await


I learned that today.

File: frontend/src/zero_coverage_report.js (hunk 5 above)
Comment (La0) [reply frontend/src/zero_coverage_report.js]: Yeap.

File: frontend/.eslintrc.yml (hunk 3 above)
Comment (La0) [reply frontend/.eslintrc.yml]: [Default settings from eslint](https://github.com/eslint/eslint/blob/4c0b70b869c16647f7af6de9d5c5479fc19f49db/lib/init/config-initializer.js#L272).
I trust those real JS dev to know what they do :smirk:

**Pull request merged**